### DISCUSSION TITLE
How i continue?

### DISCUSSION THREAD

**Discussion by Psikose** [DIGITALCRIMINALS/OnlyFans#497] [Q&A]
Someone can help?

![error](https://user-images.githubusercontent.com/108226734/175818673-1bcc5de8-e5bc-467f-9e1a-4db9c9f1d47f.jpg)

**Comment by carbon42069** [comment]
run `python updater.py` first then try again

**Comment by Tyemnata** [comment]
Try running it as: poetry run python ./start_ofd.py
